docs/developers/api.md

```diff
@@ -4,9 +4,24 @@ title: API Overview
 sidebar_label: Overview
 ---
 
+## Compatibility considerations
+
+[`scalafix-core`](https://index.scala-lang.org/scalacenter/scalafix/scalafix-core) makes the packages
+below available to built-in and external rules.
+
+Its A.B.C scheme follows [PVP](https://pvp.haskell.org/) for syntactic and semantic binary compatiblity
+and aims at keeping source compatibility across versions.
+- A or B (the major version) is increased whenever running rules built against the upcoming version
+  with an environment frozen at the current version may cause either runtime errors, or false positive
+  or  negative tree selection. A warning is issued at runtime in that case.
```

Comment:
Here too, I find the wording hard to follow.

```suggestion
- A.B (the major version) is increased whenever a binary incompatibility is introduced. Running a rule built with an older version of scalafix may cause either runtime errors, or false positive
  or  negative tree selection. A warning is issued at runtime in that case.
```

Reply:
Thanks for your suggestion. I am adding "major" (older **major** version), and updating the sentence below to follow the same pattern "Running a rule xxxx",